src/anniversary/sf33rd/Source/Game/aboutspr.c

```diff
@@ -0,0 +1,631 @@
+#include "sf33rd/Source/Game/aboutspr.h"
+#include "common.h"
+#include "sf33rd/AcrSDK/ps2/foundaps2.h"
+#include "sf33rd/Source/Game/AcrUtil.h"
+#include "sf33rd/Source/Game/CHARID.h"
+#include "sf33rd/Source/Game/DC_Ghost.h"
+#include "sf33rd/Source/Game/EFFECT.h"
+#include "sf33rd/Source/Game/MTRANS.h"
+#include "sf33rd/Source/Game/WORK_SYS.h"
+#include "sf33rd/Source/Game/bg_data.h"
+#include "sf33rd/Source/Game/color3rd.h"
+#include "sf33rd/Source/Game/debug/Debug.h"
+#include "sf33rd/Source/Game/texgroup.h"
+#include "sf33rd/Source/Game/workuser.h"
+
+// bss
+MultiTexture mts[24]; // size: 0x960, address: 0x6B4070
+MTS_OK mts_ok[24];
+WORK dmwk_moji; // size: 0x388, address: 0x6B3BC0
+WORK dmwk_kage; // size: 0x388, address: 0x6B3830
+
+// first part of rodata
+const u16 effk8k9_pattern[18] = { 36896, 36897, 36898, 36899, 36900, 36901, 36902, 36903, 36905,
+                                  36906, 36907, 36908, 36909, 36910, 36912, 36914, 36916, 36918 };
+
+const u32 judge_area_attr[17][2] = { { 2147475455, 96 }, { 2147471359, 96 }, { 2147463167, 96 }, { 2147459071, 96 },
+                                     { 2147442687, 96 }, { 2147438591, 96 }, { 2147434495, 96 }, { 2147430399, 96 },
+                                     { 2952740608, 96 }, { 2147450624, 96 }, { 2936078335, 96 }, { 2936074239, 96 },
+                                     { 2130751487, 96 }, { 2130747391, 96 }, { 2147483647, 64 }, { 2147483647, 32 },
+                                     { 2147483647, 96 } };
+
+void Init_load_on_memory_data() {
+    copy_char_base_data();
+    load_any_color(0x9CU, 0x18U);
```

Comment:
```suggestion
    load_any_color(0x9C, 0x18);
```

The same applies to other unsigned literals. They are almost never necessary